### PR TITLE
Resolves issue 4710.

### PR DESCRIPTION
**Pull request by iamchairs**
https://github.com/driftyco/ionic/issues/4710

### CODE REVIEW

File: js/views/scrollViewNative.js [hunk 1]
```diff
@@ -448,11 +448,9 @@
       var self = this;
       var container = self.__container;
 
-      container.removeEventListener('resetScrollView', self.resetScrollView);
```
Comment (iamchairs) [inline js/views/scrollViewNative.js]: This event doesn't exist on `container`.

**Comment by mlynch** [comment]
Thank you! Looks good, thanks for digging in and finding that mistake.

File: test/unit/views/scrollViewNative.unit.js [hunk 2]
```diff
@@ -28,14 +28,19 @@ describe('Scroll View', function() {
...
+    spyOn(document,'removeEventListener');
     var sv = new ionic.views.ScrollNative({
       el: sc
     });
     sv.__cleanup();
 
     expect(sc.removeEventListener).toHaveBeenCalled();
-    expect(sc.removeEventListener.callCount).toBe(4);
-    expect(sc.removeEventListener.mostRecentCall.args[0]).toBe('resetScrollView');
+    expect(sc.removeEventListener.callCount).toBe(2);
```
Comment (iamchairs) [inline test/unit/views/scrollViewNative.unit.js]: Only 2 events bound on the container. Only 2 events need to be removed.

File: test/unit/views/scrollViewNative.unit.js [hunk 3]
```diff
@@ -28,14 +28,19 @@ describe('Scroll View', function() {
...
+    expect(sc.removeEventListener.mostRecentCall.args[0]).toBe('scrollChildIntoView');
+
+    expect(document.removeEventListener).toHaveBeenCalled();
+    expect(document.removeEventListener.callCount).toBe(1);
```
Comment (iamchairs) [inline test/unit/views/scrollViewNative.unit.js]: The 3rd event is bound to the document.

**Pull request closed without merging**